packages/charts/src/utils.ts

```diff
@@ -210,3 +218,42 @@ export const generateTicks = ({
 
   return ticks;
 };
+
+export const calculateColorScale = (
+  minValue: number,
+  maxValue: number,
+  colorsArray?: string[],
+  colorMode: ColorModeType = 'shades'
+) => {
+  const colors =
```

Comment:
The type - `string[]` already inform that the array should be provided as argument. I suggest to rename it just to `colors`.